providers/cstor/cstor-block-device-pool/spc.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: openebs.io/v1alpha1
+kind: StoragePoolClaim
+metadata:
+  name: pool-name
+spec:
+  name: pool-name
+  type: blockdevice
+  poolSpec:
+    poolType: striped
+  blockDevices:
+    blockDeviceList:
```

Comment:
Shall we use jinja templates instead of yaml

Reply:
@gprasath We can make this change when we have -block-device-claim already created.